src/main/java/org/commonwl/view/workflow/WorkflowRepositoryCustom.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
```

Comment:
cwlviewer is not part of Apache, so perhaps we shouldn't say this?

Reply:
+1. Huh, I think I copied the header from another file in the project. I hope I didn't accidentally copy-paste from a file outside the project (or a file that was copied from an AL project).